client/views/actions/whowas.tpl

```diff
@@ -0,0 +1,10 @@
+<p>WHOWAS info for {{> ../user_name nick=whowas.nick}}</p>
```

Comment:
It may be better to say `Available info for disconnected {{nick}}` or something. Not a fan of showing `WHOWAS` here.

Reply:
How about we straight up use `whois` template/message type? We can change this leading line by adding extra flag in whois response like `whowas: true`.

Reply:
It's pretty big already, but since we should get more or less the same fields that also exist in whois, why not.

Reply:
That's the idea, since the fields are shared, there's little reason to use a separate template.